core/src/main/res/values-ar/strings.xml

```diff
@@ -720,8 +720,8 @@
   <!--Feed settings/information screen-->
   <string name="authentication_label">تسجيل الدخول</string>
   <string name="authentication_descr">غيّر اسم المستخدم وكلمة المرور لهذا البودكاست وحلقاته.</string>
-  <string name="feed_folders_label">مجلدات</string>
```

Comment:
Please don't change the translation files - just the English one.

Reply:
I only changed the name reference, not the value.

Reply:
Yeah, this makes your PR a lot larger, though, without a benefit. The files are overwritten anyways.